src/ImageSharp/Memory/BufferExtensions.cs

```diff
@@ -52,6 +52,18 @@ public static void Clear<T>(this IBuffer<T> buffer)
             buffer.Span.Clear();
         }
 
+        /// <summary>
+        /// Fills the contents of this buffer.
+        /// </summary>
+        /// <param name="buffer">The buffer</param>
+        /// <param name="value">The value to fill the buffer with.</param>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void Fill<T>(this IBuffer<T> buffer, T value)
```

Comment:
Does this have a caller? The above code works directly on the Span

Reply:
Removed again, used this in one of my intermediate experiments I think.